.gen/go/shared/shared.go

```diff
@@ -33318,7 +33319,10 @@ func (v *RegisterDomainRequest) ToWire() (wire.Value, error) {
 		fields[i] = wire.Field{ID: 40, Value: w}
 		i++
 	}
-	if v.EmitMetric != nil {
+	if v.EmitMetric == nil {
```

Comment:
I don't see the cadence-idl submodule update corresponding to this. Is that included ?

Reply:
The idl update was committed to repo idl with this pr https://github.com/uber/cadence-idl/pull/10

Reply:
> I don't see the cadence-idl submodule update corresponding to this. Is that included ?

The idl update was committed to repo idl with this pr uber/cadence-idl#10